icons/pytorchlightning.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>PyTorch Lightning icon</title><path d="M12 0L1.736 6v12L12 24l10.264-6V6zm-1.777 18l1.08-4.657-2.43-2.397L13.792 6l-1.083 4.665 2.417 2.384z"/></svg>
```

Comment:
@PeterShaggyNoble talking about the small differences around the outer edges, do you see in the illustration or the this attached svg?

Reply:
@Borda, the screenshot I linked to above is of the source SVG (in black) overlaid with the version in the PR (in red). After resizing it to `24` high, the source SVG is `20.5` wide whereas the PR version is `20.528` wide.

Reply:
I see now, so I ll update the linked icon and come back... :]

Reply:
Hold that thought, @Borda, didn't notice you'd updated the SVG in the PR - let me do another comparison ...

Reply:
ok, made the correct scaling, thx for catching this issue :]